src/components/LeftArrowButton.jsx

```diff
@@ -16,7 +16,7 @@ export default function LeftArrowButton(props) {
       className={`left-arrow-button ${LeftArrowClass}`}
```

Comment:
`LeftArrowClass` variable above, should be `leftArrowClass` to fit with the rest of the project's style.